Update formula forms dev guide with new attributes (#864)
SUSE/doc-susemanager: modules/salt/pages/formulas-custom.adoc

    - option2
----

$visible::

> Allows you to show a field or group if a condition is met. The condition must be written using the jexl[https://github.com/TomFrost/jexl] expression language.

Reviewer: ```suggestion
Allows you to show a field or group if a condition is met. 
You must use the jexl[https://github.com/TomFrost/jexl] expression language to write the condition.
```

> An example condition is:

Reviewer: ```suggestion
For example:
```

+
----
formValues.some_group.anouther_group.my_checkbox == true
----
+

> Absolute paths always begin with ``formValues``.

Reviewer: ```suggestion
Absolute paths must begin with ``formValues``.
```

+

> The field with this attribute associated with it will be shown only when the condition is met.

Reviewer: ```suggestion
The field with this attribute will only show if the condition is met.
```

> In this example the field will be shown only if ``my_checkbox`` is checked.

Reviewer: ```suggestion
In this example, the field will show only if ``my_checkbox`` is checked.
```

> The ability to use conditional statements is not limited to check boxes.

Reviewer: ```suggestion
You can also choose other elements for the conditional statement, such as select fields or text fields.
```

Reviewer: I think you missed this one :wink:

> It may also be used to check values of select-fields, text-fields, and similar.

Reviewer: ```suggestion
```

+

> A check box should be structured like this:

Reviewer: ```suggestion
Use this structure for a check box conditional:
```

+
----
some_group:
  $type: group

  another_group:
    $type: group

      my_checkbox:
        $type: boolean
----
+

> Relative paths can be specified using ``this.parent.value``. This references to the value of the parent.

Reviewer: ```suggestion
Specify relative paths using ``this.parent.value`` to define the value of the parent.
```

> To refer to the parent of the parent one can use ``this.parent.parent.value``.

Reviewer: ```suggestion
You can also refer to the parent of the parent, with ``this.parent.parent.value``.
```

This is mostly useful for ``edit-group``.
+
----
some_group:
  $type: group

  another_group:
    $type: group

    my_checkbox:
      $type: boolean

    my_text:
      $visible: this.parent.value.my_checkbox

  yet_another_group:
    $type: group

    my_text2:
      $visible: this.parent.parent.value.another_group.my_checkbox

----
+

> If you use multiple groups with the attribute, you can allow a user to select an option and show a completely different form, dependent upon the selected value.

Reviewer: ```suggestion
If you use multiple groups with the attribute, users can select an option that will show a completely different form, depending on the selected value.
```
